apollo-engine-ktor/build.gradle.kts

```diff
@@ -53,30 +53,26 @@ kotlin {
       }
     }
 
-    findByName("commonMain")?.apply {
-      dependencies {
-        api(libs.apollo.runtime)
-        implementation(libs.kotlinx.coroutines.core)
-        api(libs.ktor.client.core)
-        api(libs.ktor.client.websockets)
-      }
-    }
-    findByName("commonTest")?.apply {
-      dependencies {
-        implementation(libs.apollo.engine.tests)
-        implementation(libs.kotlin.test)
-        implementation(libs.kotlinx.coroutines.test)
-        implementation(libs.kotlinx.coroutines.core)
-      }
+    dependencies {
```

Comment:
Can you keep the `commonMain`/`commonTest` code?

Reply:
Isnt this already covered by the usage of `testImplementation`?